src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,741 @@
+        )
+    )
+    result = (2 * nf) * (gq1 + gq2)
+    return result
+
+
+@nb.njit(cache=True)
+def gamma_gg(N, nf):
+    r"""Compute the NLO gluon-gluon anomalous dimension.
+
+    Implements Eqn. (B.12) from :cite:`Mitov:2006wy`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    nf : int
+        No. of active flavors
+    cache : numpy.ndarray
+        Harmonic sum cache
+    is_singlet : boolean
+        True for singlet, False for non-singlet, None otherwise
+
+    Returns
+    -------
+    gamma_gg : complex
+        NLO gluon-gluon anomalous dimension
+        :math:`\gamma_{gg}^{(1)}(N)`
+
+    """
+    s1 = w1.S1(N)  # c.get(c.S1, cache, N, is_singlet)
+    s2 = w2.S2(N)  # c.get(c.S2, cache, N, is_singlet)
+    s3 = w3.S3(N)  # c.get(c.S3, cache, N, is_singlet)
+    sm1 = w1.Sm1(N, s1, is_singlet=None)  # to be removed
+    sm2 = w2.Sm2(N, s2, is_singlet=None)  # c.get(c.Sm2, cache, N, is_singlet)
+    sm3 = w3.Sm3(N, s3, is_singlet=None)  # c.get(c.Sm3, cache, N, is_singlet)
+    sm21 = w3.Sm21(N, s1, sm1, is_singlet=None)  # c.get(c.Sm21, cache, N, is_singlet)
+
+    m1tpN = 1
+
+    gg1 = (
+        nf
+        * constants.CF
+        * (
+            (
+                2
+                * (
+                    -16
+                    + 8 * N
+                    + 108 * npp(N, 2)
+                    + 162 * npp(N, 3)
+                    + 106 * npp(N, 4)
+                    + 11 * npp(N, 5)
+                    - 5 * npp(N, 6)
+                    - 2 * npp(N, 7)
+                    + 6 * npp(N, 8)
+                    + 5 * npp(N, 9)
+                    + npp(N, 10)
+                )
+            )
+            / (npp(-1 + N, 2) * npp(N, 3) * npp(1 + N, 3) * npp(2 + N, 2))
+        )
+    )
+    gg2 = (
+        constants.CA
+        * constants.CA
+        * (
+            (
+                2
+                * (
+                    -576
+                    + 240 * N
+                    + 3824 * npp(N, 2)
+                    + 1240 * npp(N, 3)
+                    + 1928 * npp(N, 4)
+                    + 8303 * npp(N, 5)
+                    + 10651 * npp(N, 6)
+                    + 6614 * npp(N, 7)
+                    + 1238 * npp(N, 8)
+                    - 1133 * npp(N, 9)
+                    - 889 * npp(N, 10)
+                    - 288 * npp(N, 11)
+                    - 48 * npp(N, 12)
+                    + 1152 * zeta2 * npp(N, 2)
+                    + 1248 * zeta2 * npp(N, 3)
+                    - 1296 * zeta2 * npp(N, 4)
+                    - 792 * zeta2 * npp(N, 5)
+                    + 876 * zeta2 * npp(N, 6)
+                    - 1368 * zeta2 * npp(N, 7)
+                    - 2340 * zeta2 * npp(N, 8)
+                    + 120 * zeta2 * npp(N, 9)
+                    + 1476 * zeta2 * npp(N, 10)
+                    + 792 * zeta2 * npp(N, 11)
+                    + 132 * zeta2 * npp(N, 12)
```

Comment:
Collect things to make them readable and more efficient. Mathematica can help you to do this, if you need.